spec/lib/crm/crm_cases_spec.rb

```diff
@@ -2,54 +2,18 @@
 
 require 'spec_helper'
 
+crm_case_user = api.crm.create_person(random_word.capitalize, random_word.capitalize).data
+crm_case = api.crm.create_case(random_word, { members: crm_case_user['id'] }).data
+crm_case_for_group_deleting = api.crm.create_case(random_word).data
+case_for_group_deleting_by_filter = api.crm.create_case(random_word).data
+
 describe '[CRM]' do
   let(:teamlab_module) { :crm }
 
-  describe '#create_person' do
-    it_behaves_like 'an api request' do
-      let(:teamlab_module) { :crm }
-      let(:command) { :create_person }
-      let(:args) { [random_word.capitalize, random_word.capitalize] }
-      let(:add_data_to_collector) { true }
-      let(:data_param) { :new_contact_ids }
-      let(:param_names) { %w[id] }
-    end
-  end
-
-  describe '#create_case' do
```

Comment:
I think we shouldnot delete this describe, but only remove usage of `@data_collector`

Because removing this describe we remove check that `create_case` method is working correctly
I know we calling it on line 6 but it can return any strange result on line 6 and we do not know if this data correct